include/openssl/sslerr.h

```diff
@@ -551,6 +555,7 @@ int ERR_load_SSL_strings(void);
 # define SSL_R_SRTP_COULD_NOT_ALLOCATE_PROFILES           362
 # define SSL_R_SRTP_PROTECTION_PROFILE_LIST_TOO_LONG      363
 # define SSL_R_SRTP_UNKNOWN_PROTECTION_PROFILE            364
+# define SSL_R_SSL3_EXT_INVALID_MAX_FRAGMENT_LENGTH       232
```

Comment:
this number is already used for `SSL_F_SSL_CTRL` don't know if that is a problem.

Reply:
The `_F_` and `_R_` numbers are entirely different namespaces so this does not matter.